tests/test_compose.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from videopython.base.compose import VideoComposer
+from videopython.base.transforms import TransformationPipeline
+from videopython.base.transitions import FadeTransition, InstantTransition
+
+
+def test_vanilla_compose(small_video):
+    videos_to_compose = small_video.split()
+    transformation_pipeline = TransformationPipeline([])
+    transition = InstantTransition()
+    composer = VideoComposer(transformation_pipeline, transition)
+    composed_video = composer.compose(videos_to_compose)
+    assert composed_video.total_seconds == 2 * small_video.total_seconds
```

Comment:
After splitting videos, test no longer valid

Reply:
Ahh, the small_video no longer existed after split, fixed now 😌